lib/juttle-engine.js

```diff
@@ -1,45 +1,28 @@
-var Base = require('extendable-base');
+'use strict';
 var _ = require('underscore');
 var express = require('express');
 var expressWs = require('express-ws');
-var bodyParser = require('body-parser');
-var jobs = require('./job-handlers');
-var paths = require('./path-handlers');
-var prepares = require('./prepare-handlers');
 var logger = require('log4js').getLogger('juttle-engine');
 
 var read_config = require('juttle/lib/config/read-config');
 var Juttle = require('juttle/lib/runtime').Juttle;
-var JuttledErrors = require('./errors');
+var JuttleExpressRoutes = require('./express-routes');
 
 var API_PREFIX = '/api/v0';
 
-var JuttleEngine = Base.extend({
+class JuttleEngine {
 
-    initialize: function(options) {
-        var config = read_config(options);
+    constructor(options) {
+
+        let config = read_config(options);
         Juttle.adapters.configure(config.adapters);
-        var self = this;
 
-        self._config_path = options.config_path;
         this._app = express();
         expressWs(this._app);
```

Comment:
I think it makes more sense to move this inside the router since it can't work without it -- you can guard against the case where the user has already registered expressWs with something like:

```
if (! app.ws) {
    expressWs(app);
}
```